lighthouse-core/index.js

```diff
@@ -62,6 +62,13 @@ module.exports = function(url, flags, configJSON) {
 
     // kick off a lighthouse run
     resolve(Runner.run(driver, {url, flags, config}));
+  })).catch(err => {
+    if (err.message.toLowerCase().includes('multiple tabs')) {
+      log.error('status', err.message);
+      return Promise.reject('');
```

Comment:
Not clear why we reject upwards with an empty message in this case, but I think I'm missing something..